### DISCUSSION TITLE
GitHub Actions workflow "pages build and deployment" missing logs from "Build with jekyll" job

### DISCUSSION THREAD

**Discussion by hpshelton** [community/community#115220] [Actions]
### Select Topic Area

Bug

### Body

When running the built-in GitHub Actions workflow "pages build and deployment", the log output from actions/jekyll-build-pages@v1 is streamed to the console while viewing the live build, but when the job completes, the log output is no longer visible. Note that the chevron to expand the log output for "Build with jekyll" is missing in [this build](https://github.com/hpshelton/hpshelton.github.io/actions/runs/8493703000/job/23268075050):

![image](https://github.com/community/community/assets/156465/d6e991b4-9f9d-4e36-9054-99df47ba5557)
 
Similarly, the log output does not appear when clicking "View raw logs"; instead a "Failed to generate URL to download logs" error occurs. 

"Download log archive" similarly is missing the logs from this task:

![image](https://github.com/community/community/assets/156465/e196b977-8d22-4c2e-9f0e-b5f953dcebae)

I have subsequent tasks that review this log output for their logic and need these logs retained for at least 24 hours.

**Comment by luisnquin** [comment]
I have the same problem too.
